cloudinit/sources/__init__.py

```diff
@@ -89,26 +89,26 @@ def process_instance_metadata(metadata, key_path='', sensitive_keys=()):
     @return Dict copy of processed metadata.
     """
     md_copy = copy.deepcopy(metadata)
-    md_copy['base64_encoded_keys'] = []
-    md_copy['sensitive_keys'] = []
+    base64_encoded_keys = []
+    l_sensitive_keys = []
```

Comment:
l_sensitive?

Reply:
was just trying to set a local variable name to avoid overriding the param that is passed in. will change the name to something useful/clear